js/ccf-app/src/endpoints.ts

```diff
@@ -103,6 +103,7 @@ export interface Request<T extends JsonCompatible<T> = any> {
   caller?: AuthnIdentity;
 }
 
+// TODO: Describe AllOfAuthnIdentity
```

Comment:
Need to do this before merging.

Worryingly it looks like these types are not tested at all currently, so this probably means moving the `multi_auth` endpoint to TS...